chichiwang/exercise-js
# Split Documentation page sections into modules

## Description
Linked to Issue: #65 
The file `pages/Documentation/index.tsx` is getting a bit large. For the sake of legibility and maintainability, the significant sections of documentation content have been broken out into their own modules:
* Starting a Project
* NPM Scripts
* Running the Application
* Project Directories
* Adding an Application Directory

## Changes
* New modules created for each significant section of documentation content, the JSX copied over from `pages/Documentation/index.tsx`
  * `pages/Documentation/StartingAProject.tsx`
  * `pages/Documentation/NPMScripts.tsx`
  * `pages/Documentation/RunningTheApplication.tsx`
  * `pages/Documentation/ProjectDirectories.tsx`
  * `pages/Documentation/AddingAnApplicationDirectory.tsx`
* The content of the modularized sections are removed from `pages/Documentation/index.tsx` and replaced with rendering the new components

## Steps to QA
* Pull down and switch to this branch
* Verify in browser that the `/documentation` page looks the same as before
  * Especially in regards to the spaces before/after links and highlighted text
